tests/scripts/loopback_snmp_acls.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+#
+# config_service_acls.sh
+#
+#  Configure service ACLs in such a way that should prevent Ansible
+#  server from connecting to running services. This should cause Ansible
+#  playbook tasks which remotely connect to thses services to fail (timeout),
+#  ensuring service ACLs are implemented correctly and actually dropping
+#  unpermitted traffic.
+#
+
+# Generate an ACL config file which allows only IP address ranges which
+# shouldn't contian the IP address of our Ansible server
```

Comment:
Script name and description incorrect. Looks like you copied this script from config_service_acls.sh but forgot to update this.

Also, is this script really necessary? I originally wrote this for testing control plane ACLs, because I needed to block SSH traffic and then unblock it again after some time to allow Ansible to SSH back into the device. You're only modifying SNMP ACLs here -- can you not simply do it via `duthost.shell()` calls rather than using a script like this? You won't sever SSH connectivity like I needed to do.

Reply:
Removed script and used ip table rules instead.